src/transformers/models/glpn/configuration_glpn.py

```diff
@@ -80,7 +80,7 @@ class GLPNConfig(PretrainedConfig):
             The dimension of the decoder.
         max_depth (`int`, *optional*, defaults to 10):
             The maximum depth of the decoder.
-        in_index (`int`, *optional*, defaults to -1):
+        head_in_index (`int`, *optional*, defaults to -1):
             The index of the features to use in the head.
```

Comment:
```suggestion
The index of the hidden state (feature) from the backbone to use in the head.
```
Maybe a little more descriptive?